src/main/java/ai/elimu/util/ml/ReadingLevelUtil.java

```diff
@@ -0,0 +1,41 @@
+package ai.elimu.util.ml;
+
+import ai.elimu.model.v2.enums.ReadingLevel;
+import org.pmml4s.model.Model;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import static ai.elimu.util.AppConstants.READING_LEVEL_CONSTANTS.*;
+
+public class ReadingLevelUtil {
+
+    public static ReadingLevel predictReadingLevel(
+            int chapterCount,
+            int paragraphCount,
+            int wordCount,
+            String modelFilePath
```

Comment:
```suggestion
            int wordCount
```

Reply:
@Aryant-Tripathi I suggest we remove the model file path as a method parameter, because we will only be using one and the same model. So the solution here could be to point to the model file path from within the `ReadingLevelUtil` itself, with the added benefit of reducing complexity in the Java classes that will be using the model for making predictions.

Reply:
@jo-elimu I guess the Utility for defining modelFilePath can be used for test cases. As I created a sample model file path for test cases i.e. model1.pmml
I thought maybe we should not use the current existing model file, that's why I paas the modelFilePath.